app/routes/api/applications.js

```diff
@@ -96,22 +98,23 @@ export const applicationHandlers = [{
   }
 },
 {
-  method: 'POST',
+  method: 'post',
   path: '/api/application/claim',
   options: {
     validate: {
-      payload: Joi.object({
-        approved: Joi.boolean().required(),
-        reference: Joi.string().valid(),
-        user: Joi.string().required()
+      payload: joi.object({
+        approved: joi.boolean().required(),
+        reference: joi.string().valid(),
+        user: joi.string().required(),
+        note: joi.string()
```

Comment:
Should this have .optional() suffixed? Is this endpoint ever called from somewhere that isn't the process that will mandate adding a note?

Reply:
optional is the default, so it's fine to use this endpoint without submitting one. Reference should maybe be required though 🤔

Reply:
weird that it has .valid(), that should be for a restricted list of values, no? Maybe just change that out to required()